.pipelines/templates/.generate-github-token.yaml

```diff
@@ -1,6 +1,8 @@
 steps:
 - bash: |
     set -euo pipefail
+    # don't echo commands - this will cause the ADO setvariable to fail in strange ways. Also, token security.
+    set +x
```

Comment:
isnt this the default ?

Reply:
Yeah, but I wanted to specify it explicitly, so the next developer is forced to read the comment.